### DISCUSSION TITLE
Suppress library hash check without disabling all checks?

### DISCUSSION THREAD

**Discussion by neilcsmith-net** [LWJGL/lwjgl3#1007] [Q&A]
I'm using LWJGL in [PraxisLIVE](https://www.praxislive.org/)  I'd really like to be able to suppress the hash check on the native libraries, but I can't seem to do this without disabling all checks.  This causes a problem on macOS, as all binaries are deep signed with the same code certificate during packaging which changes the hash.  As this is an IDE, I'd prefer not to just disable all checks to stop users seeing the error message.  Is there a good way to achieve this?

Thanks! :smile:

**Comment by Spasi** [comment]
Hey @neilcsmith-net,

Added a new option with https://github.com/LWJGL/lwjgl3/commit/7b6ba7dedba9153bc685014d8d2c2aea3f0e5602, it will be available in the first `3.3.5` snapshot.

**Reply by neilcsmith-net** [reply]
Fantastic! Thanks @Spasi
